### PR TITLE
Update easy-policy-setup-edu.md

### PR DESCRIPTION
**Pull request by hubgitsk**
Hi 

The changes in article are made due to improvements in EDU Policy Wizard. We want to publish this doc as BTS NH is starting from August. Please review and accept the changes. Let me know in case of any query. 

Thanks

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/easy-policy-setup-edu.md [hunk 1]
```diff
@@ -51,6 +51,8 @@ The wizard sets up policies based on educational institution type (**Primary or
 
 You only need to run the wizard one time. New students automatically get the Global (Org-wide default) policy definitions applied by the wizard and new staff that you add to the group you selected are automatically assigned the custom policies.
 
+Also, whenever a new feature is added to Teams, the appropriate EDU relevant default value of the policy for that feature will be automatically added to Global (Org-wide default) without requiring any admin's intervention. This helps make sure the right policies are in place to keep students secure and engaged.
```
Comment (JohanFreelancer9) [inline Teams/easy-policy-setup-edu.md:54]:
```suggestion
Also, whenever a new feature is added to Teams, the appropriate EDU relevant default value of the policy for that feature will be automatically added to the Global (Org-wide default) without requiring any admin intervention. This helps to ensure that the right policies are in place to keep students secure and engaged.
```

File: Teams/easy-policy-setup-edu.md [hunk 2]
```diff
@@ -130,9 +132,9 @@ See [Keeping students safe while using Teams for distance learning](https://supp
 
 ### Step 3: Check Message Center for policy updates
 
-Currently, the wizard applies our recommended policies when you run it. It's important to know that as new policies  become available in Teams, the Global (Org-wide default) settings for student safety aren't automatically added by the wizard. This capability will be available in a future release.
+Currently, the wizard applies our recommended policies when you run it. It's important to know that as new policies  become available in Teams, the Global (Org-wide default) settings for student safety are automatically updated by the wizard.
```
Comment (JohanFreelancer9) [inline Teams/easy-policy-setup-edu.md:135]:
```suggestion
Currently, the wizard applies our recommended policies when you run it. It's important to know that as new policies become available in Teams, the Global (Org-wide default) settings for student safety are automatically updated by the wizard. 
```

File: Teams/easy-policy-setup-edu.md [hunk 3]
```diff
@@ -130,9 +132,9 @@ See [Keeping students safe while using Teams for distance learning](https://supp
 
 ### Step 3: Check Message Center for policy updates
 
-Currently, the wizard applies our recommended policies when you run it. It's important to know that as new policies  become available in Teams, the Global (Org-wide default) settings for student safety aren't automatically added by the wizard. This capability will be available in a future release.
+Currently, the wizard applies our recommended policies when you run it. It's important to know that as new policies  become available in Teams, the Global (Org-wide default) settings for student safety are automatically updated by the wizard. 
 
-Until this capability is available, check [Message Center](https://admin.microsoft.com/AdminPortal/Home?#/MessageCenter) (in the Microsoft 365 admin center) frequently to stay up to date on new policies and policy settings in Teams. As new features become available, you may have to manually update your policies to keep your learning environment safe.
+But please do check [Message Center](https://admin.microsoft.com/AdminPortal/Home?#/MessageCenter) (in the Microsoft 365 admin center) frequently to stay up to date on new features and their policies and policy settings in Teams.
```
Comment (JohanFreelancer9) [inline Teams/easy-policy-setup-edu.md:137]:
```suggestion
Do check the [Message Center](https://admin.microsoft.com/AdminPortal/Home?#/MessageCenter) (in the Microsoft 365 admin center) frequently to stay up to date on new features and their policies and policy settings in Teams. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review. @hubgitsk - kindly commit the minor suggested changes. For information, we do not use the word "please" in docs, except in special cases which do not apply here. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by hubgitsk** [comment]
Hey Team,

Can you please expedite publishing the article. I am still seeing the old content. Teams for Education Policy Wizard to easily apply policies for safe learning - Microsoft Teams | Microsoft Docs<https://docs.microsoft.com/en-US/MicrosoftTeams/easy-policy-setup-edu?WT.mc_id=TeamsAdminCenterCSH&tabs=students%2Cstudent-settings>

This is related to EDU. As BTS Northern Hemisphere is opening, we want to make sure EDU admins get to know the modifications asap and apply policies using EDU Policy Wizard.

We want to publish the changes max by 8/3/21.

Please let me know, if there are any challenges or concerns.

Thanks
Shashank
________________________________

**Comment by JohanFreelancer9** [comment]
@scanum

**Comment by scanum** [comment]
@cichur PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by hubgitsk** [comment]
@cichur<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcichur&data=04%7C01%7CShashank.Johri%40microsoft.com%7C808b05b1efd04e5634c908d9569ebd10%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637636062098932778%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=W2eWdKQsSaaBbyQvVnLrwThn13QXx9EIfriBpludELM%3D&reserved=0> could you please merge today?
________________________________

**Comment by hubgitsk** [comment]
Hi Teams

I am still seeing old content Teams for Education Policy Wizard to easily apply policies for safe learning - Microsoft Teams | Microsoft Docs<https://docs.microsoft.com/en-US/MicrosoftTeams/easy-policy-setup-edu?WT.mc_id=TeamsAdminCenterCSH&tabs=students%2Cstudent-settings>

This is extremely urgent .

We already missed 3rd. We need to close it today.

Thanks
Shashank
________________________________

**Comment by hubgitsk** [comment]
Thanks @Cindy ***@***.***> !

I still do not see the changes somehow in the live link? Teams for Education Policy Wizard to easily apply policies for safe learning - Microsoft Teams | Microsoft Docs<https://docs.microsoft.com/en-US/MicrosoftTeams/easy-policy-setup-edu?WT.mc_id=TeamsAdminCenterCSH&tabs=students%2Cstudent-settings>
[https://docs.microsoft.com/en-us/media/logos/logo-ms-social.png]<https://docs.microsoft.com/en-US/MicrosoftTeams/easy-policy-setup-edu?WT.mc_id=TeamsAdminCenterCSH&tabs=students%2Cstudent-settings>
Teams for Education Policy Wizard to easily apply policies for safe learning - Microsoft Teams<https://docs.microsoft.com/en-US/MicrosoftTeams/easy-policy-setup-edu?WT.mc_id=TeamsAdminCenterCSH&tabs=students%2Cstudent-settings>
Learn how to use the Teams for Education policy wizard to easily apply policies for students and educators to keep your learning environment safe.
docs.microsoft.com

________________________________

**Pull request merged**